hello-world/src/main/resources/archetype-resources/src/main/resources/HelloWorldBuilder/config_fr.properties

```diff
@@ -0,0 +1,3 @@
+Name=Nom
+French=Fran\u00e7ais
```

Comment:
Comment passing by: Unicode to ascii `.properties` encoding is unnecessary. It complicates reviews IMO, making it less readable. `.properties` are in ISO-8859-1, which covers 95% of the needed french characters, including ç here and é below, so let's leave that encoding for things that actually *require* to be encoding... (€ and so on)